app/app.js

```diff
@@ -1270,6 +1270,23 @@ YUI.add('juju-gui', function(Y) {
             );
           }, this);
         }
+        // Handle the change set token if provided in the query.
+        // The change set token identifies a collections of changes required
+        // to deploy a bundle. Those changes are assumed to be already
+        // registered in the GUI server (via a ChangeSet:SetChanges request).
+        // Doing that is usually responsibility of a separate system
+        // (most of the times, it is Juju Quickstart).
+        var querystring = this.location.search.substring(1);
+        var qs = Y.QueryString.parse(querystring);
+        var changesToken = qs.changestoken;
```

Comment:
what is a changestoken? is it a bundle ID? A auth token and we're expecting to get the bundle yaml from quickstart? Is it something else? This feels a little bit like using a word to define itself and I wonder if some more concrete clarity would make this easier to come back to in a year.